src/types/koa.d.ts

```diff
@@ -0,0 +1,7 @@
+import * as router from 'koa';
+
+declare module 'koa' {
+  interface Request extends Koa.BaseRequest {
```

Comment:
Can you please add a comment? my understanding is that you're extending an already existing `Request` interface that's already part of koa. I wonder if the `extends Koa.BaseRequest` part is necessary since it's already part of koa's types definition.